### DISCUSSION TITLE
posts/life/2024/12/02/01/c6v3/

### DISCUSSION THREAD

**Discussion by giscus** [melted-mint/fuwari-edited#8] [Comments]
# posts/life/2024/12/02/01/c6v3/

test

https://meltedmint.com/posts/life/2024/12/02/01/c6v3/

**Comment by fgghf405** [comment]
C6V3이라는 발음기호를 보고 굉장히 마음에 들어 한번 써봤습니다.
매력적인 표기방법인 것 같습니다.

인정합니다 양순음 치음 권설음 등을 이런 기호로 표현하고 유성 무성 비음 등을 아래기호, 파열음 근접음 등을 위쪽기호로 표현한게 정말 인상깊더라고요.
그리고 모음표현도 굉장히 신박하다고 생각합니다.
혀의 위치에 따라 달라지는 모음기호와 입술을 오므릴때 점을 추가하는 방식도 정말 마음에 들었던것 같습니다.

**Reply by meltedmint** [reply]
제일 먼저 해석해주셨네요...!
정말 고마워요!
앞으로 제 인공언어에서,
외래어, 실제발음에 대한 발음표기방법으로 사용할 생각입니다.
현존하는 언어의 발음을 거의 다 표현이 가능해서...
